src/metabase/driver/mysql.clj

```diff
@@ -874,19 +880,16 @@
                     [table-name privileges])))
           table-names)))
 
-(defmethod driver/current-user-table-privileges :mysql
-  [_driver database]
+(defmethod sql-jdbc.sync.interface/current-user-table-privileges :mysql
+  [driver conn]
```

Comment:
Let's leave these implementations open to extension so we can do things like only getting select privileges
```suggestion
  [driver conn & {:as _args}]
```